bitnami/spark/templates/deployment-master.yaml

```diff
@@ -0,0 +1,183 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Could you confirm that, in both deployments, we need no persistence or headless services (that is, to ensure that each element in the cluster has a stable domain name)

Reply:
We do not need a headless service, and about the persistence I think not. The image does not have persistence and the `stable/spark` Helm Chart has not persistence neither.